.github/workflows/cmake_macos.yml

```
@@ -75,16 +78,17 @@ jobs:
-              -DCMAKE_CXX_FLAGS="-DPHARE_DIAG_DOUBLES=1 "
+              -DCMAKE_CXX_FLAGS="-O2 -DPHARE_DIAG_DOUBLES=1 "
 
     - name: Build
       working-directory: ${{runner.workspace}}/build
       run: cmake --build . -j 2
 
     - name: Test
-      working-directory: ${{runner.workspace}}/build
-      run: ctest -j 2 --output-on-failure
-
+      working-directory: ${{runner.workspace}}
```

actionlint (1.7.4)

property "workspace" is not defined in object type {arch: string; debug: string; environment: string; name: string; os: string; temp: string; tool_cache: string}

(expression)